aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -531,8 +576,17 @@ module aptos_framework::account {
         alice
     }
 
-/*
-TODO bring back with generic rotation capability
+    #[test(bob = @0x345)]
+    #[expected_failure(abort_code = 65544)]
+    public entry fun test_valid_offer_rotation_capability(bob: signer) acquires Account {
+        let pk = x"f66bf0ce5ceb582b93d6780820c2025b9967aedaa259bdbb9f3d0297eced0e18";
+        let alice = create_account_from_ed25519_public_key(pk);
+        create_account(signer::address_of(&bob));
+
+        let valid_signature = x"cd181d65eb31193dcf1627fc0cc04208f66e7f243facc840830eaa458b176de570f73b661c127d98bc276c5a07ab242734b4d656163a86803561c0b9d9d01d0c";
```

Comment:
Same issue as in the other PR: might be good to add instructions on how you generated this signature.

Reply:
Personally I don't think we should or need to validate that our crypto libraries are doing the right thing. I do agree that if we break something (which we aren't allowed to any more), then having gen libraries is great

Reply:
What do you mean by "gen libraries"?